components/stepper/display-modes.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Display modes
+page_title: Display modes
+description: Display modes of the Stepper for Blazor.
+slug: stepper-display-modes
+tags: telerik,blazor,stepper,display,modes
+published: True
+position: 17
+---
+
+# Display modes
+
+This article explains the Display modes that the Stepper for Blazor provides.
```

Comment:
Consider combining the two paragraphs into a single one, the current first paragraph basically repeats the title without adding meaningful new information.

Reply:
I'd prefer to keep them separated, so I refactored the article not to repeat information. I also fixed the labels image as I noticed some horizontal line.